src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/person/domain/OppfolgingstilfelleDag.kt

```diff
@@ -65,14 +69,33 @@ fun List<OppfolgingstilfelleDag>.gradertAtTilfelleEnd() =
         it.priorityOppfolgingstilfelleBit != null
     }.isGradert()
 
-fun List<OppfolgingstilfelleDag>.toOppfolgingstilfelle() =
-    Oppfolgingstilfelle(
+fun List<OppfolgingstilfelleDag>.toOppfolgingstilfelle(): Oppfolgingstilfelle {
```

Comment:
Tenker vi å alltid logge og telle dette når man lager et Oppfolgingstilfelle eller er det kun midlertidig for testing? 
Disse .to...()-metodene gjør jo vanligvis ikke annet enn å mappe fra et objekt til et annet, så føles litt rart at den også skal logge og telle. Men samtidig passet det jo fint å legge det her 😄

Reply:
Ja, vi kan godt ta bort dette etter hvert, men det er ikke så enkelt å telle opp eller finne eksempler på dette i ettertid, så tror jeg det vil være praktisk å ha denne loggingen i første omgang for å finne ut hvor stor effekten av å ta i bruk sykmelding-ny er (dvs anslå hvor mange kandidater dette resulterer i).